developers/bindings/thing-xml.md

```diff
@@ -983,6 +983,11 @@ e.g. If you have a multi-sensor that measures both temperature and humidity, the
 1. Alternatively to the above, if a Thing has multiple functions e.g. "WiZ Ceiling Fans With a Dimmable Bulb" then apply the tag that matches the **PRIMARY** function.
 e.g. `Equipment.CeilingFan` in this case.
 
+1. Sometimes an openHAB Thing is a device that commands another piece of equipment that is itself not directly visible in openHAB.
+e.g. an electronic dimmer may be hardwired on-site to a conventional light.
+Whilst there is no 100% guarantee that the Thing is actually hardwired to such target device, in such cases it is acceptable to tag the Thing (and its Channels) according to the **EXPECTED** target device.
+e.g. in the afore-mentioned example, instead of tagging the dimmer as `Equipment.ControlDevice` it is acceptable to tag it as `Equipment.LightSource`.

```

Comment:
```suggestion
e.g. in the aforementioned example, instead of tagging the dimmer as `Equipment.ControlDevice` it is acceptable to tag it as `Equipment.LightSource`.

```